task/helm-render-manifests-from-repo/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- 
+
+reviewers:
+- 
```

Comment:
are we planning to add someone? else can we remove this?

Reply:
Thanks for the review @vinamra28 - is it OK to not have an `OWNERS` file? That sounds like the quickest resolution, but I'm not sure of the process of adding to the catalog. If it's OK to remove, then I'll make that change along with the other suggestions.

Reply:
yeah, OWNERS file is not mandatory, it's safe to remove